src/arrayReverse.test.js

```diff
@@ -8,13 +8,33 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    const result = arrayReverse([]);
 
+    expect(Array.isArray(result)).toBe(true);
+  });
+
+  it(`should return an array with each string reversed`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
   });
 
   it(`should return an empty string
     if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+    expect(arrayReverse(['', ''])).toEqual(['', '']);
```

Comment:
This test case should also consider reversing the order of the strings, even if they are empty.